pkg/util/mount/mount_linux.go

```diff
@@ -367,7 +367,11 @@ func (mounter *SafeFormatAndMount) formatAndMount(source string, target string,
 			}
 
 			if fstype == "ext4" || fstype == "ext3" {
-				args = []string{"-F", source}
+				if len(mountArgs) == 0 {
+					args = []string{"-F", source}
+				} else {
+					args = append(mountArgs, source)
```

Comment:
append to `options`? otherwise we need to have `-o` in mount command to proceed the mount options.